### PR TITLE
Update maintainer status, add Will Beason

### PR DESCRIPTION
**Pull request by maxsmythe**
Re-up maintainer status for:

* sozercan
* ritazh
* smythe
* tsandall
* timothyhinrichs
* ashutosh-narkar

Swap willbeason in place of ctab

Leaving Oren to re-up when he's ready.

Signed-off-by: Max Smythe <smythe@google.com>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -4,15 +4,16 @@ The following table lists OPA project maintainers and areas of expertise in alph
...
 | Oren Shomron | @shomron | shomron@gmail.com | VMware | frameworks/constraints, gatekeeper, gatekeeper-library, cert-controller | 2020-11-13 |
-| Rita Zhang | @ritazh | rita.z.zhang@gmail.com | Microsoft | frameworks/constraints, gatekeeper, gatekeeper-library, cert-controller | 2021-03-29 |
-| Sertaç Özercan | @sozercan | sozercan@gmail.com | Microsoft | gatekeeper, gatekeeper-library, cert-controller | 2021-03-29 |
-| Tim Hinrichs | @timothyhinrichs | timothy.l.hinrichs@gmail.com | Styra | all repositories | 2021-03-29 |
-| Torin Sandall | @tsandall | torinsandall@gmail.com | Styra | all repositories | 2021-03-29 |
+| Rita Zhang | @ritazh | rita.z.zhang@gmail.com | Microsoft | frameworks/constraints, gatekeeper, gatekeeper-library, cert-controller | 2022-03-24 |
+| Sertaç Özercan | @sozercan | sozercan@gmail.com | Microsoft | gatekeeper, gatekeeper-library, cert-controller | 2022-03-24 |
+| Tim Hinrichs | @timothyhinrichs | timothy.l.hinrichs@gmail.com | Styra | all repositories | 2022-03-24 |
+| Torin Sandall | @tsandall | torinsandall@gmail.com | Styra | all repositories | 2022-03-24 |
```
Comment (tsandall) [inline MAINTAINERS.md:14]: nit: we were keeping the table in  alphabetical order (first name, last name). Could you move Will to the end? Thanks!

**Review by tsandall — COMMENTED** [review]
thanks @maxsmythe 🙏 

As a side note, once this is merged, I'd like to nominate @srenatus from Styra as a maintainer on the `opa` repo. @srenatus has been part of the community since 2018. Since then, he's shipped a number of valuable features (most recently, the `every` keyword) and has been an invaluable resource for the community when it comes to support.

**Review by sozercan — APPROVED** [review]
Thanks @maxsmythe and welcome @willbeason!

**Comment by maxsmythe** [comment]
Absolutely! Good call with the alphabetical thing, updated.

+1 to adding srenatus for OPA

**Review by willbeason — APPROVED** [review]

**Review by ritazh — APPROVED** [review]

**Pull request merged**